core/lib/spree/core/importer/order.rb

```diff
@@ -97,21 +97,66 @@ def self.create_shipments_from_params(shipments_hash, order)
           end
         end
 
-        def self.create_line_items_from_params(line_items_hash, order)
-          return {} unless line_items_hash
-          line_items_hash.each_key do |k|
-            begin
-              extra_params = line_items_hash[k].except(:variant_id, :quantity, :sku)
-              line_item = ensure_variant_id_from_params(line_items_hash[k])
-              line_item = order.contents.add(Spree::Variant.find(line_item[:variant_id]), line_item[:quantity])
-              # Raise any errors with saving to prevent import succeeding with line items failing silently.
-              if extra_params.present?
-                line_item.update_attributes!(extra_params)
-              else
-                line_item.save!
+        def self.create_line_items_from_params(line_items, order)
```

Comment:
Keep the perceived complexity. It'll be way easier to delete the code in the future.